src/main/java/frc/robot/commands/ConeCubeSwap.java

```diff
@@ -51,19 +53,28 @@ public ConeCubeSwap() {
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=REQUIRES
     }
 
-    // Called when the command is initially scheduled.
-
-    @Override //change color of CANdle if cube/cone
-    public void initialize() {
-        boolean isCube = true;
-        CANdle candle = new CANdle(0);
-        if (isCube) { // if it is a cube (not cone)
-            candle.setLEDs(255, 0, 255);// purple
-        } else { // if it is not a cube (cone)
-            candle.setLEDs(255, 255, 0);// yellow
+    // Called when the command is initially scheduled
+   @Override //change color of CANdle if cube/cone
+   public void initialize() {
+        XboxController xboxController = new XboxController(0);
+       CANdle candle = new CANdle(0);
+       if (xboxController.getYButton()) {
```

Comment:
Line 61 through 73 can be condensed into just one if(isCube) + else. Keep in mind that you should be calling commands from the OI section near the end of RobotContainer, NOT through the commands themselves. You should not be declaring the controller, something used universally, inside something as specific as a command.

(I believe Andy has a version of the code like this.)